meson.build

```diff
@@ -199,9 +199,18 @@ endif
 
 cne_conf.set('HAS_UINTR_SUPPORT', false)
 if cc.has_argument('-muintr') and cc.get_id() == 'gcc' and cc.has_header('x86gprintrin.h')
+  ret=run_command('cat', '/etc/redhat-release', check:false)
```

Comment:
Instead of doing all this, I think we can just do:
```
if cc.links('#include <x86gprintrin.h>\nint main(){_stui();return 0;}', args: '-muintr')
    cne_conf.set('HAS_UINTR_SUPPORT', true)
    add_project_arguments('-muintr', language: 'c')
endif
```

Please look at https://github.com/CloudNativeDataPlane/cndp/pull/207

Reply:
Thanks Jeff, 

Yes, I agree.

I can help to test it and we can merge #207.

Mark